readme.md

```diff
@@ -59,6 +60,7 @@ If you're interested in documenting or verifying the behaviour of the original M
 * It does not work with Minecraft Java or Bedrock edition servers
 * It does not have a survival mode (nor will such a mode be added)
 * It does not support logging in with Mojang/Minecraft accounts
+* It does not support or require Java or UWP (which isnt supported on windows 8.1 and lower) meaning it can be run on all versions without Java installed.

```

Comment:
Repeated statement. Also misconception about UWP in general, as it's foundation was actually introduced in Windows 8, and later rebranded to UWP when Windows 10 came out.